emcc.py

```diff
@@ -2918,6 +2918,11 @@ def parse_args(newargs):
       skip = False
       continue
 
+    # Support legacy '--bind' flag, by mapping to `-lembind` which now
```

Comment:
Should this be below the `.strip()` line just in case?

Reply:
I don't think so.   If you manage create a situation where there is a space at the start or end of one of your individual arguments then we should treat that is intentional.

In other words programs can expect argv to be well formed, and its up to the shell to handle any whitespace stripping.

For example, in python if you do `subprocess.call(['/bin/ls', ' --version'])` (note the extra space in the `--version` argument) then you will get:

```
/bin/ls: cannot access ' --version': No such file or directory
```

Reply:
I'm just not sure why we strip whitespace in arguments anyway - I agree that

> If you manage create a situation where there is a space at the start or end of one of your individual arguments then we should treat that is intentional.

but the comment on that line seems to suggest sometimes such space is unintentional and a result of a bug?

Reply:
Oh sorry I didn't see the line below.  My bad.  I will investigate.